pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -387,6 +394,9 @@ func (m *cgroupManagerImpl) toResources(resourceConfig *ResourceConfig) *libcont
 	}
 	if resourceConfig.Memory != nil {
 		resources.Memory = *resourceConfig.Memory
+		if resourceConfig.Swap != nil {
+			resources.MemorySwap = *resourceConfig.Memory + *resourceConfig.Swap
```

Comment:
Why do we account the regular memory as part of the swap?

Reply:
https://github.com/opencontainers/runc/blob/b1b0e7f8d9fb0650e64831b7466893bb6c046702/libcontainer/configs/cgroup_linux.go#L56-L57

In cgroup v1, `MemorySwap` means total memory usage (memory + swap).

Reply:
Maybe I need to add some comments here.

Reply:
Updated

Reply:
I see, thanks!